beginner_source/onnx/export_simple_model_to_onnx_tutorial.py

```diff
@@ -10,12 +10,6 @@
 
 **Author**: `Ti-Tai Wang <https://github.com/titaiwangms>`_, `Justin Chu <justinchu@microsoft.com>`_, `Thiago Crepaldi <https://github.com/thiagocrepaldi>`_.
 
-.. note::
```

Comment:
I think we can keep this note. All ONNX tutorial should be using the new way: ``torch.onnx.export(..., dynamo=True)``

Reply:
yes, but now that I read it, I'm thinking it's okay to keep mentioning. I would rephrase though to something like:

* ``torch.onnx.export`` is the outdated way that uses a no longer supported TorchScript and is not recommended.

Reply:
Done